frontend/src/modelInterface/SimpleForm.tsx

```diff
@@ -127,12 +127,14 @@ const SimpleForm = ({setCount}) => {
       body: formData,
     })
       .then(function (response) {
-        //handle success
+        alert("Your model is running! The model typically takes about 10 minutes per year calculated.")
```

Comment:
```suggestion
        alert("Your model is running! You must keep this tab open at this page while it runs. The model typically takes about 10 minutes per year calculated, but it may be more or less than that")
```